core/modules/layout/layout.admin.inc

```diff
@@ -812,6 +812,27 @@ function layout_content_form($form, &$form_state, Layout $layout, $renderer_name
     '#suffix' => '</div>',
   );
 
+  // Add a link to preview the page.
+  $link_path = $layout->getPath();
+  // A null path returns the site root. See https://github.com/backdrop/backdrop-issues/issues/4383
+  if (!is_null($link_path) && backdrop_valid_path($link_path)) {
+    $form['top']['#theme_wrappers'] = array('container');
+    $form['top']['#attributes']['class'] = array('edit-layout-top', 'clearfix');
+
+    $form['top']['button'] = array(
+      '#type' => 'dropbutton',
+    );
+    $form['top']['button']['#links'] = array(
+     array(
+        'title' => t('Preview page'),
```

Comment:
IMO this should be "View page", not "Preview page", since it's the real thing, not a preview.